src/frontend/src/lib/components/exchange/ExchangeTokenValue.svelte

```diff
@@ -1,18 +1,12 @@
 <script lang="ts">
-	import { nonNullish } from '@dfinity/utils';
+	import TokenExchangeBalance from '$lib/components/tokens/TokenExchangeBalance.svelte';
 	import TokenExchangeValueSkeleton from '$lib/components/tokens/TokenExchangeValueSkeleton.svelte';
 	import type { TokenUi } from '$lib/types/token';
-	import { formatUSD } from '$lib/utils/format.utils';
 
 	export let token: TokenUi;
+	$: ({ balance, usdBalance } = token);
```

Comment:
Can you declare `balance` and `usdBalance` for consistency? Did not double check but, I think we always do so.

Reply:
I don't know why I didn't... strange! changed